app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,18 @@
+
+<h2 class="secondary">Goal-Setting for the Overacheiver Hiding Deep Inside You</h2>
+<ul>
+  <% @tasks.each do |task|%>
+  <li class="task">
+    <%= link_to "#{task.name}", "/tasks/#{task.id}"%> <%if task.completed? %>
```

Comment:
Best practice probably would have this path going to `task_path(task.id)` instead of the interpolated string